### PR TITLE
Fix power sources on Centrifuge

### PR DESCRIPTION
**Pull request by nilederg**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -31,3 +31,4 @@ jar{
         include "plugin.json"
     }
 }
+targetCompatibility = JavaVersion.VERSION_1_9
```
Comment (bend-n) [inline build.gradle]: you cant do this

File: src/main/PlagueMain.java [hunk 2]
```diff
@@ -1529,7 +1530,7 @@ void loadMap(mindustry.maps.Map map) {
         });
         isSerpulo = PlagueData.serpuloCores.contains(Team.malis.cores().get(0).block);
         world.tiles.forEach(t -> {
-            if (t.build != null && t.build.block.equals(Blocks.powerSource))
+            if (t.build != null && t.build.block.equals(Blocks.powerSource) && t.team() == Team.malis)
```
Comment (bend-n) [inline src/main/PlagueMain.java]: a Tile does not have a team() method, use `t.build.team()`

**Review by bend-n — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (nilederg) [reply build.gradle]: why not

File: src/main/PlagueMain.java (hunk 2 above)
Comment (nilederg) [reply src/main/PlagueMain.java]: it does but i changed it anyway

File: build.gradle (hunk 1 above)
Comment (bend-n) [reply build.gradle]: because mindustry doesnt support this

Comment (nilederg) [reply build.gradle]: gone

**Review by bend-n — APPROVED** [review]

**Pull request merged**